examples/scripts/parser/kin.js

```diff
@@ -1,42 +1,62 @@
 
 NGL.autoLoad('data://1crnFH-multi.kin').then(function (kinemage) {
   for (let master in kinemage.masterDict) {
-    var shape = new NGL.Shape(master, {
-      pointSize: 3,
-      sizeAttenuation: false,
-      lineWidth: 3
-    })
+    var shape = new NGL.Shape(master)
 
     kinemage.dotLists.forEach(function (dotList) {
-      if (!dotList.master.includes(master)) return
-      for (var i = 0, il = dotList.position.length / 3; i < il; ++i) {
-        var i3 = i * 3
-        var x = dotList.position[ i3 ]
-        var y = dotList.position[ i3 + 1 ]
-        var z = dotList.position[ i3 + 2 ]
-        var r = dotList.color[ i3 ]
-        var g = dotList.color[ i3 + 1 ]
-        var b = dotList.color[ i3 + 2 ]
-        shape.addPoint([ x, y, z ], [ r, g, b ], dotList.label[ i ])
```

Comment:
You should actually use the shape approach here to get support for picking/hover labels

Reply:
Ah, ok, cool!  I'll give it a try.  That's my fault for not looking more closely at your recent commits.